### PR TITLE
Add repeating questions section to PHDC builder

### LINKED ISSUE #1243: Implement the GENERIC REPEATING QUESTIONS section of the PHDC body
Based on the sample HepA and Pretussis PHDCs from the GDIT team we will need to build the `GENERIC REPEATING QUESTIONS` second of the PHDC case report body.

Acceptance Criteria:

We have a `_build_generic_repeating_questions` method and associated data class updates in the `PHDCBuilder` and this method is called when we build a PHDC case report.

### PR DESCRIPTION
**Pull request by m-goggins**
# PULL REQUEST

## Summary
This PR adds a repeating questions section. For the MVP and until we know more from Texas/NBS, we are only adding Observations where `category` == `EXPOS` (exposure) to the repeating questions section. 

## Related Issue
Fixes #1243 

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/message-parser/app/phdc/builder.py [hunk 1]
```diff
@@ -342,6 +343,37 @@ def _build_social_history_info(self) -> ET.Element:
...
+            },
+        )
+        title = ET.Element("title")
+        title.text = "REPEATING QUESTIONS"
+        section.append(code)
+        section.append(title)
+
+        # add observation data to section
+        if self.input_data.repeating_questions is not None:
+            for observation in self.input_data.repeating_questions:
```
[the remark this answers is not in the document]
Comment (DanPaseltiner) [reply containers/message-parser/app/phdc/builder.py:370]: I see we have this pattern in the two other methods. I'll add a ticket to address this everywhere unless you don't think it is a good idea.

**Review by DanPaseltiner — APPROVED** [review]
LGTM! One comment below on a change I think we should make. Happy to address in another ticket though.

**Review by robertmitchellv — APPROVED** [review]
looks great; i need to add some more data to the new sample bundle based on this!

File: containers/message-parser/app/phdc/builder.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-goggins) [reply containers/message-parser/app/phdc/builder.py:370]: I think it's a good idea. I can just address it here for all of the methods.

**Pull request merged**